packages/theme/components/FormComponents/EmailInput.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
+    <SfInput
+      @click="e => $emit('click', e)"
+      @input="e => $emit('input', e)"
+      v-bind="{...$attrs, ...$props}"
+      :class="{'disable-input': disabled}"
+      :valid="!errors[0]"
+      :errorMessage="errors[0]"
+    />
+  </div>
+</template>
+
+<script>
+import {
+  SfInput
+} from '@storefront-ui/vue';
+
+export default {
+  name: 'EmailInput',
+  props: {
+    disabled: {
+      type: Boolean,
+      default() {
+        return false;
+      }
+    },
+    label: {
+      type: String,
+      default() {
+        return '';
+      }
+    },
+    name: {
+      type: String,
+      default() {
+        return 'email';
+      }
+    },
+    value: {
+      type: String
+    },
+    errors: {
+      type: Array
+    }
+  },
+  components: {
```

Comment:
Let's keep components right below the name property to stay consistent across other component definitions.

Reply:
Got it!